### DISCUSSION TITLE
clarify what true looks like in output

### DISCUSSION THREAD

**Discussion by rdp** [dbeaver/dbeaver#15308] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**

DBeaver display of boolean is slightly confusing.

**Describe the solution you'd like**

If I have a boolean column, I'd like it more obvious.

ex: postgres

```
select true;
```

It shows a [v] for the output pane.  
A "v"?  What type is that?  What am I seeing?

Suggestion/feature request for UI:

[true] and [false] or possibly show a "real-er" checkmark...

Cheers!

**Comment by scmehle** [comment, marked as answer]
This feature does exist, but you have to dig through the preferences a bit.

Click on the "Window" menu and choose "Preferences". Navigate to the editors section, and there is a section to set how Booleans display. Screenshot attached for reference.
![image](https://user-images.githubusercontent.com/9370225/151252583-30d2579a-2f03-4a0a-a1ac-064a72d0fd5a.png)

**Comment by rdp** [comment]
That's interesting, mine on OS X just shows up as a "v" like [v]"...

**Reply by scmehle** [reply]
You may also need to check the version of DBeaver you are using. I'm using the latest community version, and I think this has changed a bit in the last few releases.

**Comment by LonwoLonwo** [comment]
And you can change it in the Preferences.

![2022-01-27 11_07_10-Preferences](https://user-images.githubusercontent.com/45152336/151317222-0a7c9915-5b92-4741-addf-d6a3d9ff9c2a.png)

**Reply by rdp** [reply]
Interesting.  Seems (at least on mac) the default is [v] which is actually pretty confusing.  I guess consider this a feature request to make the default "anything else" (checkbox, or [true] or what have you).  Thanks for your kind attention :)

**Comment by awcator** [comment]
[v] is pretty confusing. Please make true/false as the default

**Reply by bitwisebro** [reply]
Agree, this should be changed to true/false for better readability

**Comment by MathanGeurtsen** [comment]
This setting has since changed location. 


The right place is now:

Window > Preferences > User Interface > Miscellaneous
![image](https://github.com/user-attachments/assets/e79f9a85-f67e-4dc1-97df-c49f0b69c792)

(sorry for new comment, but this question ranks high on google results)

**Comment by RichardJECooke** [comment]
Why is `true` and `false` not the default after two years? I just wasted twenty minutes trying to understand where [v] came from in my client's database :(
